openquake/qa_tests_data/event_based_risk/case_master/expected/avg_losses-mean.csv

```diff
@@ -1,8 +1,8 @@
 asset_ref,taxonomy,lon,lat,business_interruption,contents,nonstructural,occupants:float32,structural,business_interruption_ins,contents_ins,nonstructural_ins,occupants_ins:float32,structural_ins
-a3,"tax1",-122.57000,38.11300,9.40465E+01,5.16592E+02,7.74888E+02,1.88093E-03,9.28790E+01,2.32312E+00,2.32313E+00,2.32313E+00,0.00000E+00,5.11875E-01
-a2,"tax2",-122.11400,38.11300,7.20282E+02,7.63771E+03,1.00958E+04,1.44056E-02,1.10447E+03,9.88750E+00,9.97500E+00,9.97500E+00,0.00000E+00,3.06687E+00
-a5,"tax1",-122.00000,37.91000,3.01209E+02,1.68611E+03,2.45068E+03,6.02419E-03,2.42266E+02,7.70437E+00,7.70437E+00,7.70437E+00,0.00000E+00,6.25625E-01
-a4,"tax3",-122.00000,38.00000,4.65241E+02,2.88366E+03,4.50574E+03,9.30482E-03,5.44473E+02,9.58562E+00,8.71937E+00,9.69938E+00,0.00000E+00,1.53125E+00
-a1,"tax1",-122.00000,38.11300,1.69277E+03,7.82866E+03,1.38052E+04,3.38553E-02,6.96359E+03,9.97500E+00,9.97500E+00,9.97500E+00,0.00000E+00,7.96250E+00
-a6,"tax2",-122.00000,38.22500,6.73931E+02,5.31437E+03,7.82135E+03,1.34786E-02,9.87915E+02,9.67312E+00,9.94000E+00,9.94000E+00,0.00000E+00,2.11750E+00
-a7,"tax1",-121.88600,38.11300,5.53284E+02,3.30581E+03,4.77473E+03,1.10657E-02,9.27300E+02,9.91375E+00,9.91375E+00,9.91375E+00,0.00000E+00,2.98375E+00
+a3,"tax1",-122.57000,38.11300,9.80829E+01,5.39780E+02,8.09670E+02,1.96166E-03,9.51180E+01,2.32312E+00,2.32312E+00,2.32313E+00,0.00000E+00,5.11875E-01
```

Comment:
This is due to the different management of the epsilons. I checked that without epsilons the new calculator produces exactly the same numbers as the old ones.